### PR TITLE
Small updates to the JS core course

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Make certain sentences more readable.
Update code identifiers
Fix grammar mistakes

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: javascript/core/types-ii/truthy-falsy-gotchas.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ false == "" //true
 
 ### Null
 
-Null is also one to beware of as behaves strangely:
+Null is also one to beware of as it behaves strangely:
```
Comment (nem035) [inline javascript/core/types-ii/truthy-falsy-gotchas.md:32]: "Strangely" doesn't feel appropriate. Perhaps we should use "unexpectedly"?

```suggestion
Null is also one to beware of as it behaves unexpectedly:
```

**Review by nem035 — DISMISSED** [review]
:+1:

**Comment by nem035** [comment]
Don't forget to add a changelog entry

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/array-methods/apply-map-function-to-array-items.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/array-methods/apply-map-function-to-array-items.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/array-methods/filtering-items-out-of-an-array.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/array-methods/filtering-items-out-of-an-array.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/array-methods/how-to-merge-two-arrays.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/array-methods/how-to-merge-two-arrays.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/array-methods/shuffle-an-array-using-sort-and-random.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/array-methods/shuffle-an-array-using-sort-and-random.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/arrays/setting-the-length-of-an-array.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/arrays/setting-the-length-of-an-array.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/errors/catch-is-block-scoped.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/errors/catch-is-block-scoped.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/errors/detect-an-error-type.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/errors/detect-an-error-type.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/errors/exceptions.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/errors/exceptions.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/flow-control/if-statement.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/flow-control/if-statement.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-i/access-a-function-s-arguments-as-an-array.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-i/access-a-function-s-arguments-as-an-array.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-i/functions-can-be-declared-after-use.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-i/functions-can-be-declared-after-use.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-i/setting-default-values-with-short-circuiting.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-i/setting-default-values-with-short-circuiting.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/anonymous-functions.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/anonymous-functions.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/check-if-a-function-exists.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/check-if-a-function-exists.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/functions-are-objects.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/functions-are-objects.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/functions-return-undefined-by-default.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/functions-return-undefined-by-default.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/iife-pattern.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/iife-pattern.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-iii/bind.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-iii/bind.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-iii/changing-a-functions-context-with-fn-call-object.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-iii/changing-a-functions-context-with-fn-call-object.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-iii/difference-between-call-and-apply-methods.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-iii/difference-between-call-and-apply-methods.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/functions-iii/function-call-method.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-iii/function-call-method.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/in-depth-ii/javascript-compilation.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/in-depth-ii/javascript-compilation.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/in-depth-ii/meaning-of-polyfill.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/in-depth-ii/meaning-of-polyfill.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/in-depth-ii/method-chaining.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/in-depth-ii/method-chaining.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/in-depth-ii/prototype-methods.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/in-depth-ii/prototype-methods.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/js-history/infix-operators-are-left-associative.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/js-history/infix-operators-are-left-associative.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/js-history/strict-mode.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/js-history/strict-mode.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/numbers/base-conversion.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/base-conversion.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/numbers/inaccuracy-of-binary-floating-point-format.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/inaccuracy-of-binary-floating-point-format.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/numbers/nan-is-a-number.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/nan-is-a-number.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/numbers/parsing-numbers-and-ieee-754.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/parsing-numbers-and-ieee-754.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/objects/converting-to-and-from-json.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/objects/converting-to-and-from-json.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/objects/deleting-properties.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/objects/deleting-properties.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/objects/hasownproperty-method.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/objects/hasownproperty-method.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/objects/object-part-1.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/objects/object-part-1.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/easily-generate-a-random-hex-color.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/easily-generate-a-random-hex-color.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/format-currency-in-javascript.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/format-currency-in-javascript.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/quick-way-to-format-integers-with-a-fixed-number-of-digits.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/quick-way-to-format-integers-with-a-fixed-number-of-digits.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/use-the-apply-function-to-get-the-min-or-max-of-an-array.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/use-the-apply-function-to-get-the-min-or-max-of-an-array.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/using-the-double-tilde.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/using-the-double-tilde.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-ii/avoid-using-with.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-ii/avoid-using-with.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-ii/milliseconds-since-epoch.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-ii/milliseconds-since-epoch.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/recipes-ii/performing-a-function-at-timed-intervals.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-ii/performing-a-function-at-timed-intervals.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/scope/scope-and-strict-mode.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/scope/scope-and-strict-mode.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/types-ii/be-careful-when-using-typeof.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/types-ii/be-careful-when-using-typeof.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/types-ii/truthy-falsy-gotchas.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/types-ii/truthy-falsy-gotchas.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/types-ii/you-can-t-handle-the-truth.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/types-ii/you-can-t-handle-the-truth.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)
- [ ] [enkidevs/curriculum/javascript/core/types/types-undefined-and-null.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/types/types-undefined-and-null.md?ref=c8a20010fd24e8dbba341a9f10fadeb4da7b3023)

**Review by nem035 — APPROVED** [review]

**Pull request merged**